Core/Core/Login/LoginWebViewController.swift

```diff
@@ -27,9 +27,16 @@ class LoginWebViewController: UIViewController, LoginWebViewProtocol {
     }()
 
     var presenter: LoginWebPresenter?
+    var env: AppEnvironment!
```

Comment:
`let env = AppEnvironment.shared`

I don't think we should be passing around AppEnvironment structs anymore, since even in tests we are overwriting AppEnvironment.shared.